File: dataset_reduzido/avatar/Python/Code/codeforces_59_A.py
txt = input ( )
cu = 0
cl = 0
for z in txt :
    if z.islower ( ) :
        cl = cl + 1
    else :
        cu = cu + 1
if cu > cl :
    out = txt.upper ( )
else :
    out = txt.lower ( )
print ( out )

File: output/LLaMa_repaired_1/avatar/Python/Java/codeforces_59_A.java
import java.util.Scanner;

public class codeforces_59_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String txt = scanner.nextLine().trim();
        scanner.close();

        int cu = 0;
        int cl = 0;
        for (char z : txt.toCharArray()) {
            if (Character.isLowerCase(z)) {
                cl++;
            } else if (Character.isUpperCase(z)) { // Check for valid input
                cu++;
            }
        }

        if (cu > cl) {
            System.out.println(txt.toUpperCase());
        } else {
            System.out.println(txt.toLowerCase());
        }
    }
}